src/config.ts

```diff
@@ -0,0 +1,17 @@
+import {ClientOptions, Intents} from 'discord.js';
+import {LogLevel} from '@sapphire/framework';
+import {fetchPrefix} from './utils/discordUtil';
+
+export const DEFAULT_PREFIX = '$';
+
+export const CLIENT_OPTIONS: ClientOptions = {
+  intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_INTEGRATIONS, Intents.FLAGS.GUILD_MESSAGES, Intents.FLAGS.DIRECT_MESSAGES],
+  defaultPrefix: DEFAULT_PREFIX,
+  caseInsensitiveCommands: true,
+  caseInsensitivePrefixes: true,
+  logger: {
+    level: LogLevel.None,
+  },
```

Comment:
To clarify, this is the discord logger right?

Reply:
Or our main logger?

Reply:
If this is our main logger, this should be at INFO, otherwise nothing will actually get logged

Reply:
It's the discord logger, because we don't use it.